docs/advanced-features/custom-error-page.md

```diff
@@ -55,3 +55,23 @@ export default Page
 ```
 
 The `Error` component also takes `title` as a property if you want to pass in a text message along with a `statusCode`.
+
+## Default Static 404 Page
+
+When you don't need to create a custom `_error` page and would prefer to use the default Next.js provided one, we will automatically apply the static optimization to this file during a production build and use it when deploying on [Now](https://zeit.co) or with `next start`.
```

Comment:
I got confused here because the heading says 404 but the paragraph doesn't even mention 404. I think we should say "404" explicitly in this paragraph?

Reply:
> Good point, I reworked it using the above mentioned structure. Let me know if this seems better